docusaurus/docs/packages/pkg/client/pkg.md

```diff
@@ -113,21 +112,278 @@ txctx -."tx query".-> chain
+  if err := cmd.ExecuteContext(context.Background()); err != nil {
+    log.Fatal(err)
+  }
+}
+
+func init() {
+  // IMPORTANT!: In order to take advantage of cosmos-sdk helpers which take
+  // the cobra command object  as an input, some flags MUST be registered and
+  // populated as expected by such helpers.
+  // See:
+  //   - https://pkg.go.dev/github.com/cosmos/cosmos-sdk@v0.50.3/client#GetClientContextFromCmd
+  //   - https://pkg.go.dev/github.com/cosmos/cosmos-sdk@v0.50.3/client/tx#NewFactoryCLI
+  cmd.Flags().String(sdkflags.FlagNode, "", "Register the default Cosmos node flag, which is needed to initialize the Cosmos query and tx contexts correctly.")
+  cmd.Flags().String(sdkflags.FlagGRPC, "", "Register the default Cosmos node grpc flag, which is needed to initialize the Cosmos query context with grpc correctly.")
+}
+
+func runCommand(cmd *cobra.Command, args []string) error {
+  // NOTE: interstitial `err` returns are ignored for brevity.
+  ctx := context.Background()
+
+  // Use the websocket URL of a trusted pocket network node.
+  pocketNodeWebsocketURL, _ := url.Parse("ws://localhost:35567/websocket")
+
+  deps, _ := config.SupplyConfig(
+    ctx, cmd,
+    config.NewSupplyEventsQueryClientFn(pocketNodeWebsocketURL),
+  )
+  supplierClient, _ := supplier.NewSupplierClient(deps)
+
+  // NOTE: merkle root hash must be valid.
+  merkleRootHash := []byte("some merkle root hash")
+
+  // NOTE: session header must be valid.
+  sessionHeader := sessiontypes.SessionHeader{}
+
+  return supplierClient.CreateClaim(ctx, sessionHeader, merkleRootHash)
+}
+```
+
+### Advanced Usage
+
+```go
+// TODO(@bryanchriswhite #159): Example illustrating advanced features or edge cases of the package.
+```
+
+## Usage (Transaction Client)
 
-### Basic Example
+### Basic Example (CLI command)
 
 ```go
-// TODO: Code example showcasing the use of TxClient or any other primary interface.
+package main
+
+import (
+  "context"
+  "log"
+  "net/url"
+  "time"
+
+  "cosmossdk.io/depinject"
+  sdkclient "github.com/cosmos/cosmos-sdk/client"
+  sdkflags "github.com/cosmos/cosmos-sdk/client/flags"
+  sdktx "github.com/cosmos/cosmos-sdk/client/tx"
+  "github.com/spf13/cobra"
+  "github.com/spf13/pflag"
+  "github.com/stretchr/testify/require"
+
+  "github.com/pokt-network/poktroll/pkg/client/block"
+  "github.com/pokt-network/poktroll/pkg/client/events"
+  "github.com/pokt-network/poktroll/pkg/client/tx"
+  "github.com/pokt-network/poktroll/pkg/deps/config"
+  apptypes "github.com/pokt-network/poktroll/x/application/types"
+)
+
+// Arbitrary timeout duration for the transaction to complete.
+const txTimeoutDuration = 10 * time.Second
+
+var (
+  // GRPC URL of trusted pocket network node, used for querying chain state.
+  queryNodeGRPCURLFlag string
+  // GRPC URL of trusted pocket network node, used for submitting transactions to the network.
+  txNodeGRPCURLFlag string
+  signingKeyNameFlag string
+  cmd               = &cobra.Command{RunE: runCommand}
```

Comment:
NIT: Can we add comments to these - the identation of line 220 `cmd = ...` bothers me 😅